src/types.ts

```diff
@@ -2,7 +2,7 @@ import { Contract } from "@ethersproject/contracts"
 import { ContractInterface, ethers } from "ethers"
 
 export type Config = {
-    hyphenBaseUrl: object,
+    hyphenBaseUrl: Record<'test' | 'staging' | 'prod', string>,
```

Comment:
I think we should create a separate type like 
`type EnvironmentType = 'test' | 'staging' | 'prod'`
and use it everywhere, that would allow use to easily add support for new environments if needed in future

Reply:
@ankurdubey521 good idea! Added it in 49683bfea6d6f2fab48835de68ec4bd9f8135bf5